### PR TITLE
Mingw build fixes part1

### PR DESCRIPTION
**Pull request by astrand**
This pull request contains unobtrusive fixes for building on MinGW/GCC systems. This includes 
avoding mixed case includes and library names etc.

### CODE REVIEW

**Comment by astrand** [comment]
See also #4930.

**Comment by akallabeth** [comment]
@freerdp-bot test

File: client/Windows/CMakeLists.txt [hunk 1]
```diff
@@ -78,6 +78,7 @@ endif()
 
 set(${MODULE_PREFIX}_LIBS ${${MODULE_PREFIX}_LIBS} freerdp-client)
 set(${MODULE_PREFIX}_LIBS ${${MODULE_PREFIX}_LIBS} winpr freerdp)
+set(${MODULE_PREFIX}_LIBS ${${MODULE_PREFIX}_LIBS} msimg32)
 target_link_libraries(${MODULE_NAME} ${${MODULE_PREFIX}_LIBS})
```
Comment (hardening) [inline client/Windows/CMakeLists.txt]: can you explain why adding this library in all cases ?

File: client/Windows/wf_client.c [hunk 2]
```diff
@@ -783,7 +783,7 @@ static int freerdp_client_focus_out(wfContext* wfc)
 	return 0;
 }
 
-static int freerdp_client_set_window_size(wfContext* wfc, int width, int height)
+int freerdp_client_set_window_size(wfContext* wfc, int width, int height)
 {
```
Comment (hardening) [inline client/Windows/wf_client.c]: what's wrong this static declaration ?

Comment (akallabeth) [reply client/Windows/wf_client.c]: @hardening there is a `FREERDP_API` declaration for this in the header

**Pull request merged**